lib/cloudapp/cli/collections.rb

```diff
@@ -0,0 +1,104 @@
+require "http"
+require "pry"
+require "terminal-table"
+
+module CloudApp
+  module CLI
+    class Collections
+      COLLECTION_FIELDS = ["id", "name", "shared", "is_public", "created_at", "updated_at"]
+      ITEM_FIELDS = ["id", "slug", "name", "item_type", "view_counter", "favorite", "share_url", "created_at", "updated_at"]
+
+      def initialize(token)
+        @BASE_URL = "https://my.cl.ly/v4/collections"
+        @HTTP = HTTP.auth("Bearer #{token}")
+          .with(accept: "application/json", content_type: "application/json")
+          .accept(:json)
+      end
```

Comment:
@macsrok you should probably down-case instance variable names. Also, adding an `attr_reader` for `http`, would help you access directly without `@`.